Description: Patch makefile to be more specific when doing "find", to avoid it
 trying to search-replace-rename files in the debian/ directory during build.
Author: Stig Sandbeck Mathisen <ssm@debian.org>
Bug-Debian: http://bugs.debian.org/536987
Forwarded: no
--- a/Makefile
+++ b/Makefile
@@ -133,7 +133,7 @@
 build: build-stamp
 
 build-stamp:
-	@for file in `find . -type f -name '*.in'`; do			\
+	@for file in `find . -maxdepth 1 -type f -name '*.in'; find server node monkeywrench -type f -name '*.in'`; do			\
 		destname=`echo $$file | sed 's/.in$$//'`;		\
 		echo Generating $$destname..;				\
 		mkdir -p build/`dirname $$file`;			\

